website/docs/r/shield_protection.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "aws"
+page_title: "AWS: aws_shield_protection"
+sidebar_current: "docs-aws-resource-shield-protection"
+description: |-
+  Enables AWS Shield Advanced for a specific AWS resource.
+---
+
+# aws_shield_protection
+
+Enables AWS Shield Advanced for a specific AWS resource.
+The resource can be an Amazon CloudFront distribution, Elastic Load Balancing load balancer, AWS Global Accelerator accelerator, Elastic IP Address, or an Amazon Route 53 hosted zone.
+
+## Example Usage
+
+### Create protection
+
+```hcl
+variable "name" {
```

Comment:
Since the `variable` and `aws_eip` tags have no bearing on the actual functionality of the `aws_shield_protection` resource, we should remove them from the example. 👍